tests/sync/file.cpp

```diff
@@ -20,6 +20,7 @@
 
 #include "shared_realm.hpp"
 #include "sync/sync_manager.hpp"
+#include "util/test_utils.hpp"
```

Comment:
It is used (some functionality was split off from sync_test_utils.hpp).

This include should be grouped with sync_test_utils.hpp and not where it is, though. The current position suggests it's a part of the library being tested rather than a thing that's part of the tests themselves.